### ISSUE TITLE
Delete link preview from the flame list

### ISSUE THREAD

**Issue by madr1c** [david1602/1337#11]
I think it's a bit odd to see a link preview below the flame list especially if it contains multiple links.

My suggestion is that we should try to remove it.

**Comment by david1602** [comment]
I don't think we can enable or disable preview of telegram, that's a telegram feature. We can look into it, though. I guess you mean the response list, but this affects anything the bot posts anyway

**Comment by david1602** [comment]
The [sendMessage](https://core.telegram.org/bots/api#sendmessage) function seems to have a property called `disable_web_page_preview`, which could be utilized here.

**Comment by david1602** [comment]
Should be resolvable by adding the property [here](https://github.com/david1602/node-telegram-bot-api/blob/master/src/telegram.js#L614)
